CONTRIBUTING.md

```diff
@@ -214,20 +214,18 @@ More tests can be added to this folder to test specific functionality, or new te
 Docker alternative:
 
 1. Install [Docker](https://www.docker.com/get-started)
-2. Run `docker-compose up -d` from the command line
-3. View last results in `spec/test_results.log`
+2. Run `docker-compose up` from the command line
 
-### Creating new test builds or fixing an existing build
-
-Sometimes a change will be made that intends to change the stats garnered by PoB, which will break our tests.
-1. Add the new build XML (if applicable) to the `TestBuilds` folder
-2. Run `busted --lua=luajit -r generate` to generate a LUA file that contains the current stats of that build
-3. Run `busted --lua=luajit` and the tests should pass
-
-Docker alternative:
-
-1. Add the new build XML (if applicable) to the `TestBuilds` folder
-2. Run `docker-compose up -d` to generate a LUA file that contains the current stats of that build and run the tests
+### Debugging tests
+When running tests with a docker container it is possible to use emmylua for debugging. Paste in the following right under `function launch:OnInit()` in `./src/Launch.lua`:
+```lua
+package.cpath = package.cpath .. ";/usr/local/bin/?.so"
```

Comment:
As far as I understand the FHS, you need to put that EmmyLua library in `/usr/local/lib/` instead. `/usr/local/bin/` is for executable (binary) files, while `emmy_core.so` is a shared object (a library). See https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s09.html#purpose24

Reply:
That's true. According to FHS emmy_core.so should go into /usr/local/lib/ but for some reason it's setup to go into bin in EmmyLua's cmake install target: https://github.com/EmmyLua/EmmyLuaDebugger/blob/2b2485d2a4bf6fce80aa05d7f93cf1bfad58a1e8/emmy_core/CMakeLists.txt#L27-L31

I didn't dig into why it was that way. Calling make install puts it there:
https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/8394e2f824f6f7fdea032852d03278605c89bceb/Dockerfile#L33